### PR TITLE
Add issue search [summary] command

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
Add a command to find issues by partial summary.

### CODE REVIEW

**Comment by keepcosmos** [comment]
Thanks for PR, I wrote some reviews @maciej-szlosarczyk

**Comment by maciej-szlosarczyk** [comment]
Hey @keepcosmos, I don't see any other comments than the one above. Where did you lave those?

**Comment by keepcosmos** [comment]
@maciej-szlosarczyk Sorry, I did not start review

File: lib/terjira/issue_cli.rb [hunk 1]
```diff
@@ -109,6 +109,12 @@ def assign(*keys)
       show(issue.key_value)
     end
 
+    desc 'search [SUMMARY]', 'Search for an issue by summary'
+    def search(*keys)
+      search_term = client_class.search(summary: keys[0])
```
Comment (maciej-szlosarczyk) [inline lib/terjira/issue_cli.rb]: I'm not sure that's a good idea. Most command-line utilities treat arguments separated with a space as 2 different ones. For example:

```bash
touch "file name" # creates file named 'file name'
touch file name # creates 2 files: 'file' and 'name'
```

have different effects and I'm not sure we would want to go against that standard.

Comment (keepcosmos) [inline lib/terjira/issue_cli.rb]: You are right, I agree.

Comment (maciej-szlosarczyk) [inline lib/terjira/issue_cli.rb]: I also just checked, you can also use `jira issue search query\ string` and it will read as one argument.

**Comment by keepcosmos** [comment]
Thanks, @maciej-szlosarczyk, please rebase your commits

**Comment by keepcosmos** [comment]
Thanks @maciej-szlosarczyk  ❤️

**Comment by keepcosmos** [comment]
I just release your commit v 0.2.7

**Pull request merged**